hub/core/storage/s3.py

```diff
@@ -322,6 +326,7 @@ def __setstate__(self, state):
         self.token = state[9]
         self.loaded_creds_from_environment = state[10]
         self.read_only = state[11]
+        self.profile_name = state[12]
```

Comment:
Is it important that all of the elements of `state` are accounted for?  If so, then consider an assignment like the following:
```
(
    self.root,
    self.aws_access_key_id,
    ...
) = state
```